src/pages/hirek.tsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+/** @jsx jsx */
+
+import { Grid, jsx } from 'theme-ui';
 
 import { NewsArticleTeaser } from '../components/NewsArticleTeaser';
 
 export default function NewsPage(): JSX.Element {
-  return <NewsArticleTeaser />;
+  return (
+    <Grid
+      sx={theme => ({
+        display: 'grid',
+        gridColumnGap: '36px',
+        gridRowGap: '40px',
+        gridTemplateColumns: 'repeat(auto-fill, 500px)',
```

Comment:
Itt is érdemes inkább `ch` mértékegységet alkalmazni. Sőt, így a `NewsArticleTeaser` komponensben érdemes is elhagyni a  `maxWidth: 500` kényszert, hiszen itt a `Grid` ezt megoldja és ezáltal későbbi változtatás esetén elég lesz itt átírni az értéket.